### PR TITLE
README.md doc main page updates with better rephrases

### PR DESCRIPTION
**Pull request by vidyavemulapalli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,17 +26,17 @@ This framework is ideal for QA professionals, developers, and business analysts
 
 - **User-Friendly for All Roles**: This framework is not just for QA Automation professionals. Developers, Manual QA, and Business Analysts can also contribute to end-to-end testing, promoting collaboration across different departments and roles.
 
-- **Utility Functions**: Simplifies common actions and assertions, such as clicking buttons, filling forms, and checking elements. It also includes inbuilt methods for conditional statements and maintains a default LoadState across applications.
+- **Utility Functions**: Simplifies the task of identifying elements and performing routine actions and verifications, such as clicking buttons, inputting data into forms, and verifying elements. Additionally, it provides pre-built functions for handling conditional statements and ensures a consistent default LoadState across the applications.
```
Comment (vasu31dev) [inline README.md:29]: @vidyavemulapalli The old one looks simple. Can you rephrase this?

**Review by vasu31dev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vidyavemulapalli) [reply README.md]: @vasu31dev Updated as per review comment in the commit https://github.com/vasu31dev/playwright-ts-template/pull/2/commits/e6fea251ec5dfb3dbc9aff12699cc9a9e9ff2406

Please review. Thanks

**Review by vasu31dev — APPROVED** [review]
Approved. Thank you
